src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/setup_environment.py

```diff
@@ -36,67 +31,69 @@ def read_integers_from_string(string):
     return re.findall("\d+", string)
 
 
-def get_lower_version_from_python_specification(lower_specification):
-    """Take a lower specification and convert it to the lowest authorized python version."""
-    lower_version = ".".join(read_integers_from_string(lower_specification))
-    if lower_specification.startswith(">") and "=" not in lower_specification:
-        lower_version_splitted = lower_version.split(".")
-        lower_version_splitted[-1] = str(int(lower_version_splitted[-1]) + 1)
-        lower_version = ".".join(lower_version_splitted)
-    return lower_version
+def get_version_from_python_specification(specification):
+    """Read the version."""
+    return ".".join(read_integers_from_string(specification))
 
 
-def get_upper_version_from_python_specification(upper_specification):
-    """Take a upper specification and convert it to the highest authorized python version."""
-    upper_version = ".".join(read_integers_from_string(upper_specification))
-    if upper_specification.startswith("<") and "=" not in upper_specification:
-        upper_version_splitted = upper_version.split(".")
-        upper_version_splitted[-1] = str(int(upper_version_splitted[-1]) - 1)
-        upper_version = ".".join(upper_version_splitted)
-    return upper_version
+def get_sign_from_python_specification(specification):
+    """Read the mathematical symbol expressing a constraint on the version."""
+    first_part = specification.split(".")[0]
+    sign = "".join([i for i in first_part if not i.isdigit()])
+    if sign == "":
+        sign = "=="
+    return sign
```

Comment:
This may be simplified to:

```suggestion
    sign = "".join([i for i in first_part if not i.isdigit()])
    return sign if sign != "" else "==":
```